packages/block-editor/src/components/url-popover/link-viewer.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import {
+	ExternalLink,
+	IconButton,
+} from '@wordpress/components';
+import { safeDecodeURI, filterURLForDisplay } from '@wordpress/url';
+
+function LinkViewerUrl( { url, urlLabel, className } ) {
```

Comment:
Based on the file name, this probably should have been named `LinkViewer`. In any case, this specific naming doesn't adhere to capitalization guidelines, and it should at the very least be `LinkViewerURL`. Thankfully it doesn't seem to be part of the public API?

https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#abbreviations-and-acronyms